packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -54,6 +53,51 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any')
   };
 }
 
+/**
+ * Hook to get active patient orders.
+ *
+ * @param patientUuid The UUID of the patient whose active orders should be fetched.
+ */
+export function useActivePatientOrders(patientUuid: string) {
+  const { data: allOrders, error, isLoading, isValidating, mutate } = usePatientOrders(patientUuid);
+
+  const activeOrders = useMemo(
+    () => (allOrders ? allOrders.filter((order) => !order.autoExpireDate && !order.dateStopped) : null),
+    [allOrders],
+  );
+
+  return {
+    data: activeOrders,
+    error,
+    isLoading,
+    isValidating,
+    mutate,
+  };
+}
+
+/**
+ * Hook to get past patient orders.
+ *
+ * @param patientUuid The UUID of the patient whose past orders should be fetched.
+ */
+export function usePastPatientOrders(patientUuid: string) {
+  const { data: allOrders, error, isLoading, isValidating, mutate } = usePatientOrders(patientUuid);
+  const { data: activeOrders } = useActivePatientOrders(patientUuid);
+
+  const pastOrders = useMemo(
+    () => (allOrders && activeOrders ? allOrders.filter((order) => !activeOrders.includes(order)) : null),
```

Comment:
The filter here wouldn't work as expected. It would probably be better to:

1. Create a list of all active order UUIDs
2. Filter out any orders whose UUID is in the active order UUIDs